envoy/api/v2/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +81,17 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Target percentage of requests managed by this HTTP connection manager that will be traced
+    // after all other checks have been applied (force tracing, sampling, etc.). Defaults to 100%.
+    // This variable is a direct analog for the variable named 'global_enabled' in the :ref:`HTTP
+    // Connection Manager <config_http_conn_man_runtime>`.
+    Percent sampling = 3;
+
+    // Target percentage of requests managed by this HTTP connection manager that will be randomly
+    // traced. Defaults to 100%. This variable is a direct analog for the variable of the same name
+    // in the :ref:`HTTP Connection Manager <config_http_conn_man_runtime>`.
```

Comment:
I think it would be useful to add a note comparing the two fields. It's not entirely obvious to me how they differ.

Reply:
agreed

Reply:
I was similarly confused. I attempted a bit of clarification based on my read of the existing documentation and code. Hopefully my attempt has added clarity. Please let me know what you think.